### PR TITLE
 Add robb edge drivers ROB_200-004-0, ROB_200-011-0, ROB_200-011-0

### PR DESCRIPTION
**Pull request by zwave-mke**
@lelandblue sorry for the circumstances. We have cleaned up the branch and will move the other drivers to another branch to keep them clean.

### CODE REVIEW

File: drivers/SmartThings/zigbee-button/fingerprints.yml
Comment (dljsjr) [inline drivers/SmartThings/zigbee-button/fingerprints.yml:141]: nit: No newline at end of file.

File: drivers/SmartThings/zigbee-switch/profiles/light-level-power-energy.yml
Comment (dljsjr) [inline drivers/SmartThings/zigbee-switch/profiles/light-level-power-energy.yml:22]: nit: No newline at end of file

File: drivers/SmartThings/zigbee-button/src/zigbee-multi-button/init.lua
Comment (dljsjr) [inline drivers/SmartThings/zigbee-button/src/zigbee-multi-button/init.lua:17]: Both of these `requires` are unused, and should be removed.

**Review by dljsjr — APPROVED** [review]
Just a few style comments. LGTM otherwise.

File: drivers/SmartThings/zigbee-switch/src/robb/init.lua
Comment (greens) [inline drivers/SmartThings/zigbee-switch/src/robb/init.lua:40]: you could just change your `added` method to set this field to 1000000 and avoid repeating the defaults here

File: drivers/SmartThings/zigbee-switch/src/robb/init.lua
Comment (greens) [inline drivers/SmartThings/zigbee-switch/src/robb/init.lua:53]: you use the same unit for both of the attributes handled here? In our defaults I think we expect mW from instantaneous demand

File: drivers/SmartThings/zigbee-button/fingerprints.yml
Comment (zwave-mke) [reply drivers/SmartThings/zigbee-button/fingerprints.yml:141]: changed from CRLF to LF

File: drivers/SmartThings/zigbee-switch/profiles/light-level-power-energy.yml
Comment (zwave-mke) [reply drivers/SmartThings/zigbee-switch/profiles/light-level-power-energy.yml:22]: changed from CRLF to LF

File: drivers/SmartThings/zigbee-button/src/zigbee-multi-button/init.lua
Comment (zwave-mke) [reply drivers/SmartThings/zigbee-button/src/zigbee-multi-button/init.lua]: done

File: drivers/SmartThings/zigbee-switch/src/robb/init.lua
Comment (zwave-mke) [reply drivers/SmartThings/zigbee-switch/src/robb/init.lua]: simplified, cause it's not really needed.

File: drivers/SmartThings/zigbee-switch/src/robb/init.lua
Comment (zwave-mke) [reply drivers/SmartThings/zigbee-switch/src/robb/init.lua]: simplified too

**Review by greens — APPROVED** [review]

**Comment by zwave-mke** [comment]
@greens commits squashed. Sorry for the delay ...

**Comment by zwave-mke** [comment]
@greens commits squashed. Sorry for the delay ...

**Comment by zwave-mke** [comment]
@greens commits squashed. Sorry for the delay ...

**Comment by greens** [comment]
@zwave-mke I think we're waiting on UL to get back to us before the merge, but thanks for keeping this PR nice and tidy.

**Pull request merged**
